convert2rhel/pkgmanager/handlers/yum/__init__.py

```diff
@@ -186,19 +225,24 @@ def _perform_operations(self):
                 # package. This is an inconsistency that could lead to packages
                 # being outdated in the system after the conversion.
                 if can_update:
+                    loggerinst.info("Package %s will be updated.", pkg)
```

Comment:
Just a reminder that we can get rid of this later.

We don't need the info log for update/reinstall/downgrade.

Reply:
I will remove it from this PR. But I'm wondering if this cannot be used for better debugging of the yum transaction. If yes, then it could be part of https://issues.redhat.com/browse/RHELC-999 What do you think? @r0x0d

Reply:
So, I guess this is not related. The log message we used here to debug is mainly for us to identify which package is being added to the transaction and the operation related to the package. 

What the ticket https://issues.redhat.com/browse/RHELC-999 asks for is to capture error messages and have a better handling for that. We could potentially solve it by improving our callback classes and try to capture and output better messages there. 

Also, the errors about dependency solving only happen when we reach the part where try to resolve the dependencies. 

---

Said that, there might be one benefit to log those packages as a `DEBUG`. That would help us to understand the operation tied to the package, potentially saving us some time in the analysis when a report is opened. 

But I don't think we need to leave this for now, again, improving our callback classes might make more sense than placing those logs here.